Makefile

```diff
@@ -461,6 +461,10 @@ test-all: test-build ## Run everything in test/.
 test-all-valgrind: test-build
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=debug,release --valgrind
 
+.PHONY: test-all-suites
+test-all-suites: | bench-addons-build
```

Comment:
Doesn't this also need more prequisites for the addons and n-api tests to pass?

Reply:
Would a good way to test for completeness of the prerequisites be `make dist-clean && make test-all-suites`?

Reply:
Yes, I'd expect that to work.